archivers/upx/Portfile

```diff
@@ -47,6 +50,7 @@ depends_lib         port:ucl port:zlib
 
 post-configure {
     reinplace "s|CXXFLAGS += -fno-delete-null-pointer-checks||" ${worksrcpath}/src/Makefile
+    reinplace "s|CXXFLAGS_NO_DELETE_NULL_POINTER_CHECKS ?= -fno-delete-null-pointer-checks||" ${worksrcpath}/src/Makefile
```

Comment:
Can you explain why this is needed?

Reply:
@l2dy because of https://github.com/upx/upx/blob/devel/src/Makefile#L87

Somewhen since last release `-fno-delete-null-pointer-checks` was moved to `CXXFLAGS_NO_DELETE_NULL_POINTER_CHECKS`